### ISSUE TITLE
Infinite Loop of references

### ISSUE THREAD

**Issue by iStryker** [RESTful-Drupal/restful#493]
If you expose an entity reference field, there is no infinite loop.

If you expose an entity reference field, and use a resource of itself or another,  it creates an infinite loop if one entity (1) points to another (2), and the another (2) points back (1).   Similarly it creates an infinite loop if an entity (3) point to either (1) or (2)

**Comment by e0ipso** [comment]
This is a tricky one. Can you think of any feasible suggestion?

**Comment by amitaibu** [comment]
> Can you think of any feasible suggestion?

Yeah, this is something I didn't want to get into when first developed the feature- we could "remember" the entity that rendered in a static cache, and if it appears again, we can force the `full_view` to FALSE.

However, I think that it's ok to avoid this complexity, and let the implementors set the `full_view` by their on logic. With that said, I guess PRs are welcome ;)
